doc/api/child_process.md

```diff
@@ -332,6 +335,9 @@ changes:
 * `modulePath` {string} The module to run in the child.
 * `args` {string[]} List of string arguments.
 * `options` {Object}
+  * `advancedSerialization` {boolean} Use more flexible serialization for
```

Comment:
It doesn't seem super likely, but what if we want to support another serialization type in the future (or even worse, something like user defined types). Would making this a string that defaults to `'json'` be more future proof?

Reply:
@cjihrig Done! I’ve left the name of the new option value as `advanced`, but if you have better suggestions, I’d also take them.